search-ui/src/interfaces/corp-type.ts

```diff
@@ -0,0 +1,403 @@
+      colinInd: true,
+      corpClass: CorpClass.SOC,
+      shortDesc: 'SOCIETY',
+      fullDesc: 'Society'
+    }, {
+      corpTypeCd: CorpTypeCd.SOCIETY_BRANCH,
+      colinInd: false,
+      corpClass: CorpClass.OT,
+      shortDesc: 'SOCIETY BRANCH',
+      fullDesc: 'Society Branch'
+    }, {
+      corpTypeCd: CorpTypeCd.SOLE_PROP,
+      colinInd: true,
+      corpClass: CorpClass.FIRM,
+      shortDesc: 'SOLE PROP',
+      fullDesc: 'BC Sole Proprietorship'
+    }, {
+      corpTypeCd: CorpTypeCd.TRUST,
+      colinInd: false,
+      corpClass: CorpClass.OT,
+      shortDesc: 'TRUST',
+      fullDesc: 'Trust'
+    }, {
+      corpTypeCd: CorpTypeCd.TRAMWAYS,
+      colinInd: false,
+      corpClass: CorpClass.OT,
+      shortDesc: 'TRAMWAYS',
+      fullDesc: 'Tramways'
+    }, {
+      corpTypeCd: CorpTypeCd.BC_ULC_COMPANY,
+      colinInd: true,
+      corpClass: CorpClass.BC,
+      shortDesc: 'BC ULC COMPANY',
+      fullDesc: 'BC Unlimited Liability Company',
+      numberedDesc: 'Numbered Unlimited Liability Company'
+    }, {
+      corpTypeCd: CorpTypeCd.ULC_CO_1860,
+      colinInd: true,
+      corpClass: CorpClass.BC,
+      shortDesc: 'ULC CO 1860',
+      fullDesc: 'ULC CO 1860'
+    }, {
+      corpTypeCd: CorpTypeCd.ULC_CO_1862,
+      colinInd: true,
+      corpClass: CorpClass.BC,
+      shortDesc: 'ULC CO 1862',
+      fullDesc: 'ULC CO 1862'
+    }, {
+      corpTypeCd: CorpTypeCd.ULC_CO_1878,
+      colinInd: true,
+      corpClass: CorpClass.BC,
+      shortDesc: 'ULC CO 1878',
+      fullDesc: 'ULC CO 1878'
+    }, {
+      corpTypeCd: CorpTypeCd.ULC_CO_1890,
+      colinInd: true,
+      corpClass: CorpClass.BC,
+      shortDesc: 'ULC CO 1890',
+      fullDesc: 'ULC CO 1890'
+    }, {
+      corpTypeCd: CorpTypeCd.ULC_CO_1897,
+      colinInd: true,
+      corpClass: CorpClass.BC,
+      shortDesc: 'ULC CO 1897',
+      fullDesc: 'ULC CO 1897'
+    }, {
+      corpTypeCd: CorpTypeCd.XPRO_COOP,
+      colinInd: false,
+      corpClass: CorpClass.OT,
+      shortDesc: 'XPRO COOP',
+      fullDesc: 'Extraprovincial Cooperative Assocation'
+    }, {
+      corpTypeCd: CorpTypeCd.XPRO_LL_PARTNR,
+      colinInd: true,
+      corpClass: CorpClass.FIRM,
+      shortDesc: 'XPRO LL PARTNR',
+      fullDesc: 'Extrapro Limited Liaility Partnership'
+    }, {
+      corpTypeCd: CorpTypeCd.XPRO_LIM_PARTNR,
+      colinInd: true,
+      corpClass: CorpClass.FIRM,
+      shortDesc: 'XPRO LIM PARTNR',
+      fullDesc: 'Extraprovincial Limited Partnership'
+    }, {
+      corpTypeCd: CorpTypeCd.XPRO_SOCIETY,
+      colinInd: true,
+      corpClass: CorpClass.SOC,
+      shortDesc: 'XPRO SOCIETY',
+      fullDesc: 'Extraprovincial Society'
+    }
+  ]
+  
+  /**
+   * Given corp type code, returns the corp info object.
+   * @param cd the corp type code to get
+   * @returns the corp info object (or undefined if not found)
+   */
+  export function GetCorpInfoObject (cd: CorpTypeCd): CorpInfoIF {
```

Comment:
and then these I would expect to see in the utils in like a corp-info-helper or something. What do you think?